hack/chains/config.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+# A wrapper for kubectl patch so we can easily inspect and modify
+# tekton chains configuration.
+#
+# For reference:
+#   https://github.com/tektoncd/chains/blob/main/docs/config.md
+#   https://github.com/tektoncd/chains/blob/main/pkg/config/config.go#L99
+#
+# See also ./gitops-sync.sh. You need to disable gitops automatic
+# self healing before you can modify the configuration.
+#
+
+CHAINS_CONFIG="configmap/chains-config -n tekton-chains"
+
+case "$1" in
+
+  get )
+    # Show the current config
+    kubectl get $CHAINS_CONFIG -o yaml | yq e .data -
+
+    ;;
+
+  default )
+    # This matches the current gitops config
+    # Should work for the kaniko demo
+    $0 '{
```

Comment:
You can use ${BASH_SOURCE[0]} instead of $0 but I couldn't see much advantage, so I went with $0 for brevity and tidiness.